tor/strings/responses.py

```diff
@@ -9,8 +9,16 @@
 )
 
 claim_success = (
-    'The post is yours! Best of luck and thanks for helping!\n\nPlease respond '
-    'with "done" when complete so we can check this one off the list!'
+    'The post is yours! Best of luck and thanks for helping!\n\n {} Please '
```

Comment:
This line will have 2 spaces before "Please" if there's nothing provided. Here you can just take the spaces out on either side of the {}.

Reply:
:+1: